### DISCUSSION TITLE
翻译错误

### DISCUSSION THREAD

**Discussion by blueair5** [YiiGuxing/TranslationPlugin#1912] [General]
使用 Vscode 的时候，报 `翻译失败，error_code=52003` 错误？ 想请教如何解决，thanks 。

**Comment by YiiGuxing** [comment]
VScode？这个插件并没有VScode版本的。。。。是不是搞错了？

**Reply by blueair5** [reply]
我在 idea 装了一个，然后在 vscode 插件搜到了一个叫translation，和这个插件长得很像，后来我发现可能不是一个东西。:cry: 。这个插件很好用。

**Reply by blueair5** [reply]
昨天想写回答来着，这个教室的网太差了，抱歉。😢

**Reply by 3339543290** [reply]
> VScode？这个插件并没有VScode版本的。。。。是不是搞错了？

大佬，能不能出一个vscode版本的，现在因为ai，基于vscode的ai开发工具比较多，比如：kiro、反重力等
